src/flb_utils.c

```diff
@@ -47,7 +47,12 @@ extern struct flb_aws_error_reporter *error_reporter;
 #include <openssl/rand.h>
 #endif
 
-static char *flb_utils_replacement_character = "\xEF\xBF\xBD";
+#define FLB_UTILS_REPLACE_FRAGMENT_START_LEN 31
+#define FLB_UTILS_REPLACE_FRAGMENT_END_LEN 4
+
+static char *flb_utils_replace_fragment_start
+    = "\xEF\xBF\xBD"" corrupted utf-8 sequence : ";
```

Comment:
Could you make this a single constant character string?

Reply:
Do you mean:
1. adding the const keyword to this cstring declaration/definition?
That sounds good.
2. changing this to a Macro definition
If changed to a macro definition,  we'll need to instantiate an actual string later before copying bytes over 1 by one: https://github.com/fluent/fluent-bit/pull/4297/files#diff-d7e2a365ad467f9c9eb8283ff56105fa3a08e7fb55690a61b7137c7704a95263R803-R804
This isn't a problem since I can use a static char inside the function (but it's just adding more code for style?).